seacrowd/sea_datasets/glotstorybook/glotstorybook.py

```diff
@@ -0,0 +1,143 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses, TASK_TO_SCHEMA, SCHEMA_TO_FEATURES
+
+_CITATION = """\
+@inproceedings{kargaran2023glotlid,
+    title     = {{GlotLID}: Language Identification for Low-Resource Languages},
+    author    = {Kargaran, Amir Hossein and
+                Imani, Ayyoob and
+                Yvon, Fran{\c{c}}ois
+                and Sch{\"u}tze, Hinrich},
+    booktitle = {The 2023 Conference on Empirical Methods in Natural Language Processing},
+    year      = {2023},
+    url       = {https://openreview.net/forum?id=dl4e3EBz5j}
+}
+"""
+
+_DATASETNAME = "glotstorybook"
+_DESCRIPTION = """\
+The GlotStoryBook dataset is a compilation of children's storybooks from the Global
+Storybooks project, encompassing 174 languages organized for machine translation tasks. It
+features rows containing the text segment (text number), the language code, and the file
+name, which corresponds to the specific book and story segment. This structure allows for
+the comparison of texts across different languages by matching file names and text numbers
+between rows.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/cis-lmu/GlotStoryBook"
+_LICENSE = f"""{Licenses.OTHERS.value} | \
+We do not own any of the text from which these data has been extracted. All the files are
+collected from the repository located at https://github.com/global-asp/. The source
+repository for each text and file is stored in the dataset. Each file in the dataset is
+associated with one license from the CC family. The licenses include 'CC BY', 'CC BY-NC',
+'CC BY-NC-SA', 'CC-BY', 'CC-BY-NC', and 'Public Domain'. We also license the code, actual
+packaging and the metadata of these data under the cc0-1.0.
+"""
```

Comment:
It could be `OTHERS` but since all possible values lies on `CC` family license, do you think we shd assign it to CC, @SamuelCahyawijaya @akhdanfadh ?

Reply:
@sabilmakbar : I don't think that would be a good idea. As each type of CC license has their own restrictions. So, we cannot just combine them into a single CC.

Reply:
I agree with Samuel. If people only refer to the dataset metadata from our dataloader, it could lead to misrepresentation and misunderstandings about the licensing. However, since there's already a `Licenses` feature included in the dataset, that should provide adequate clarification on the specific type of CC license applicable to each file.

Reply:
my suggestion would look like this
```suggestion
_LICENSE = f"""{Licenses.CC.value} | \
We do not own any of the text from which these data has been extracted. All the files are
collected from the repository located at https://github.com/global-asp/. The source
repository for each text and file is stored in the dataset. Each file in the dataset is
associated with one license from the CC family. The licenses include 'CC BY', 'CC BY-NC',
'CC BY-NC-SA', 'CC-BY', 'CC-BY-NC', and 'Public Domain'. We also license the code, actual
packaging and the metadata of these data under the cc0-1.0.
"""
```

rather than this:
```python
_LICENSE = f"""{Licenses.CC.value}
```

Reply:
I just noticed that the value for CC is `Creative Commons license family (cc)`; I previously thought it referred to a different license.

I'm unsure of its original purpose and whether `Licenses.CC` will ever be utilized. However, Sabil's change could be a way to use it, unless we prefer everything to follow a template format, i.e., using either a single license or a custom license template.

Reply:
Hi @akhdanfadh @sabilmakbar @SamuelCahyawijaya, let's use `Licenses.OTHERS.value` since the license is custom-made for this dataset. I'm afraid making it `CC` would be misleading, so let's just use `OTHERS`.

Reply:
okay then, will approve this one.